src/main/java/org/fundacionjala/coding/dennis/Movie.java

```diff
@@ -0,0 +1,73 @@
+package org.fundacionjala.coding.dennis;
+
+/**
+ *
+ */
+public abstract class Movie {
+    private String title;
+    private PriceCode priceCode;
+
+    /**
+     *
+     * @param title of the movie.
+     */
+    public Movie(final String title) {
+        this.title = title;
+    }
+
+    /**
+     *
+     * @return price.
+     */
+    public PriceCode getPriceCode() {
+        return getpriceCode();
+    }
+
+    /**
+     *
+     * @param arg price.
+     */
+    public void setPriceCode(final PriceCode arg) {
+        setpriceCode(arg);
+    }
+
+    /**
+     *
+     * @return title.
+     */
+    public String getTitle() {
+        return title;
+    }
+
+    /**
+     *
+     * @return price.
+     */
+    public PriceCode getpriceCode() {
+        return priceCode;
+    }
+
+    /**
+     *
+     * @param priceCode price.
+     */
+    public void setpriceCode(final PriceCode priceCode) {
+        this.priceCode = priceCode;
+    }
+
+    /**
+     *
+     * @param nDays days rented.
+     * @return Frequent points bonus.
+     */
+    public int returnFrequentPoints(int nDays) {
+        return 1;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this method and declare a constant for this value. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS3400)

Reply:
Please create a constant for that.